### DISCUSSION TITLE
Full Scryfall set code coverage

### DISCUSSION THREAD

**Discussion by tylertyssedal** [andrewgioia/keyrune#253] [General]
Scryfall has an exhaustive list of all sets and correlated set codes, including promos. 

For example, Dragons of Tarkir Promos are listed as PDTK. While this set uses the same icon as Dragons of Tarkir (DTK), programatically removing the P from a set code as there are valid set codes that begin with P and have a length of four (such as Arena League 1996, PARL).

**Comment by csuhta** [comment]
This can be fully-automated with our API if you want to do that. If you fetch <https://api.scryfall.com/sets?pretty=true>, you can see both the set code and the actual file name for the set symbol we use.

**Comment by icobg123** [comment]
would love to see this resolved

**Comment by andrewgioia** [comment]
Moving this (and all older v4 issues) to discussions!
